.eslintrc.js

```diff
@@ -15,8 +16,14 @@ module.exports = {
         "sourceType": "module"
     },
     "plugins": [
-        "react"
+        "react",
+        "jest"
     ],
     "rules": {
+        "jest/no-disabled-tests": "warn",
+        "jest/no-focused-tests": "error",
+        "jest/no-identical-title": "error",
+        "jest/prefer-to-have-length": "warn",
+        "jest/valid-expect": "error"
     }
```

Comment:
Thank you so much for adding these. Jest is more annoying than it helps. Having a well configured eslint file is really handy.

Reply:
No problem!